cfg_mgmt/util.py

```diff
@@ -494,25 +494,46 @@ def process_cfg_queue_and_persist_in_repo(
     return True
 
 
+def _policy_violation_for_issue_type(issue_type: str) -> cmm.CfgStatusEvaluationAspects:
```

Comment:
I just checked the `cfg_mgmt` package and found there is a `reporting.py` submodule. please move reporting-related code from `cfg_mgmt/util.py` to `cfg_mgmt/reporting.py`

considering the existing reporting-functions (e.g. [this one](https://github.com/gardener/cc-utils/blob/master/concourse/steps/scan_container_images.py#L46), I think it is not (fully) adequate (or at least not symmetrical/inconsistent) to add the instrumenting code into our reporting-module. a better place might be step-template (which you already touched in this PR).